src/ProductBundle/DependencyInjection/SonataProductExtension.php

```diff
@@ -348,4 +352,14 @@ protected function registerSeoParameters(ContainerBuilder $container, array $con
             $container->setParameter(sprintf('sonata.product.seo.product.%s', $key), $value);
         }
     }
+
+    private function configureStringExtension(ContainerBuilder $container): void
+    {
+        if (!$container->hasDefinition('twig.extension.string') || !is_a($container->getDefinition('twig.extension.string')->getClass(), StringExtension::class)) {
+            $definition = new Definition(StringExtension::class);
```

Comment:
Will this work if our other bundles will register the same `StringExtension::class` service?

Reply:
It should work, becouse result will be the same.